quickfixj-core/src/main/java/quickfix/Session.java

```diff
@@ -385,6 +385,11 @@
      */
     public static final String SETTING_ALLOW_POS_DUP_MESSAGES = "AllowPosDup";
 
+    /**
+     * Option so that the session does not remove PossDupFlag (43) and OrigSendingTime (122) information when sending.
```

Comment:
> session does not remove PossDupFlag (43) and OrigSendingTime (122) information when sending

I guess its some particular usecase and the option itself doesn't know anything about these fields